packages/web-client/app/components/card-pay/create-merchant-workflow/merchant-customization/index.hbs

```diff
@@ -31,20 +32,92 @@
+            @onInput={{this.onMerchantNameInput}}
+            @invalid={{this.merchantNameValidationMessage}}
+            @errorMessage={{this.merchantNameValidationMessage}}
+            @required={{true}}
+            autocomplete="off"
+            autocorrect="off"
+            autocapitalize="off"
+            spellcheck="false"
+            {{on "blur" this.validateMerchantName}}
+          />
+        </CardPay::LabeledValue>
+        <CardPay::LabeledValue
+          @fieldMode="edit"
+          @label="Merchant ID"
+          data-test-merchant-customization-merchant-id-field
+        >
+          <CardPay::CreateMerchantWorkflow::MerchantCustomization::ValidationStateInput
+            @state={{this.merchantIdInputState}}
+            @value={{this.merchantId}}
+            @onInput={{this.onMerchantIdInput}}
+            @errorMessage={{this.merchantIdValidationMessage}}
+            @helperText="This will be used to uniquely identify your merchant in Cardstack products and cannot be changed after your merchant is created."
+            autocomplete="off"
+            autocorrect="off"
+            autocapitalize="off"
+            spellcheck="false"
+            {{on "blur" this.validateMerchantId}}
+          />
+        </CardPay::LabeledValue>
+        <CardPay::LabeledValue
+          @fieldMode="edit"
+          @label="Custom color"
+          data-test-merchant-customization-color-field
+        >
+          {{!-- template-lint-disable require-input-label --}}
+          <div class="merchant-customization__color">
+            <input
+              type="color"
+              value={{this.merchantBgColor}}
+              class="merchant-customization__color-input"
+              {{on "input" this.onMerchantBgColorInput}}
+            >
+            {{this.merchantBgColor}}
+          </div>
+        </CardPay::LabeledValue>
+        <CardPay::LabeledValue @label="Manager">
+          <CardPay::AccountDisplay
+            @wrapped={{true}}
+            @address={{this.layer2Network.walletInfo.firstAddress}}
+            data-test-merchant-customization-manager-address
+          />
+        </CardPay::LabeledValue>
+      </CardPay::FieldStack>
     {{/if}}
   </ActionCardContainer::Section>
   <Boxel::ActionChin
     @state={{if @isComplete "memorialized" "default"}}
-    data-test-layout-customization
+    @disabled={{or (not this.canSaveDetails) @frozen}}
   >
     <:default as |d|>
       <d.ActionButton
-        {{on "click" @onComplete}}
+        data-test-merchant-customization-save-details
+        {{on "click" this.saveDetails}}
       >
         Save Details
       </d.ActionButton>
     </:default>
+    <:memorialized as |m|>
+      <m.ActionButton
+      data-test-merchant-customization-edit
+        {{on "click" @onIncomplete}}
+      >
+        Edit
+      </m.ActionButton>
+    </:memorialized>
   </Boxel::ActionChin>
-</ActionCardContainer>
\ No newline at end of file
+</ActionCardContainer>
```

Comment:
[EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) should eliminate this kind of spurious diff 🤓